Assets/scss/components/_list.scss

```diff
@@ -0,0 +1,99 @@
+.list{
```

Comment:
@Satyam1RP I don't see container max-width in entire code, container max-width will be responsible to bind your content in a particular width so on large screen the content of div won't outside of that boundaries.
Cc:- @mathrik  @msharmaRP @jvaishnavrp